pkg/tfbridge/info.go

```diff
@@ -72,6 +73,9 @@ type SchemaInfo struct {
 	MaxItemsOne *bool                  // to override whether this property should project as a scalar or array.
 }
 
+// Transformer is given the option to transform a value in situ before it is processed by the bridge.
```

Comment:
Perhaps add a comment that it is expected that the Transformer be idempotent (the identity function on any value that is a non-error output)?

Reply:
Great idea. I added a comment and pushed:

```
// Transformer is given the option to transform a value in situ before it is processed by the bridge. This
// transformation must be deterministic and idempotent, and any value produced by this transformation must
// be a legal alternative input value. A good example is a resource that accepts either a string or
// JSON-stringable map; a resource provider may opt to store the raw string, but let users pass in maps as
// a convenience mechanism, and have the transformer stringify them on the fly. This is safe to do because
// the raw string is still accepted as a possible input value.
```